Mail/Components/Buttons/Custom Buttons/LoadMoreButton.swift

```diff
@@ -36,6 +36,14 @@ struct LoadMoreButton: View {
         return !currentFolder.isHistoryComplete && currentFolder.lastUpdate != nil
     }
 
+    init?(currentFolder: Folder?) {
```

Comment:
This comes from the fact `ThreadListViewModel` deals with frozen objects (that can be invalidated).
Maybe this needs further redesign ? We probably need to discuss this together.